operations/mimir-mixin/alerts/alerts.libsonnet

```
@@ -640,6 +640,25 @@
             ||| % $._config,
           },
         },
+        {
+          alert: $.alertName('RulerRemoteEvaluationFailing'),
+          expr: |||
+            100 * (
+            sum by (%s) (rate(cortex_request_duration_seconds_count{route="/httpgrpc.HTTP/Handle", status_code!~"5..", %s}[5m]))
```

I guess you to track the rate of 5xx errors, right?
```suggestion
            sum by (%s) (rate(cortex_request_duration_seconds_count{route="/httpgrpc.HTTP/Handle", status_code=~"5..", %s}[5m]))
```

---

done @ 13a267c8baf9e1facfcbd699647163bc3d4a0bf1